packages/deploy-utils/src/utils/git/index.ts

```diff
@@ -41,13 +41,14 @@ const getCommitMessages = (
 ) => {
   const gitRoot = getGitRoot();
   const appDir = resolve(gitRoot, "apps", app);
+  const packageDir = resolve(gitRoot, "packages");
   const excludeDir = resolve(
     gitRoot,
     "apps",
     app === "spruce" ? "parsley" : "spruce",
   );
   const commitMessages = execTrim(
-    `git log ${fromCommit}..${toCommit} --oneline -- ${appDir} '!${excludeDir}'`,
+    `git log ${fromCommit}..${toCommit} --oneline -- ${appDir} ${packageDir} '!${excludeDir}'`,
```

Comment:
Does this have the intented effect? i.e. everything except the excludeDir?
```suggestion
    `git log ${fromCommit}..${toCommit} --oneline -- '!${excludeDir}'`,
```

Reply:
Unfortunately not
![image](https://github.com/user-attachments/assets/9cea1620-e847-4259-af09-aa783f91ffe4)
I believe you need to provide a path to include when trying to limit output in git operations otherwise git will only perform the exclusion but not actually include anything.

 In [this example](https://stackoverflow.com/questions/5685007/making-git-log-ignore-changes-for-certain-paths) they use the `.` to include all files and then exclude ones using the `!` operator.

Reply:
Any reason we shouldn't also use `. !${excludeDir}`?

Reply:
That would include all files in the repo including those that aren't actually used in the apps. For example a change to the README would be included as a commit for the deploy.